Crash on opening Google Maps

While working on #10281 I've stumbled upon another crash with current nightly in the emulator:

```
2021-03-28 10:52:07.276 31555-31555/cgeo.geocaching E/AndroidRuntime: FATAL EXCEPTION: main
    Process: cgeo.geocaching, PID: 31555
    java.lang.RuntimeException: Unable to start activity ComponentInfo{cgeo.geocaching/cgeo.geocaching.maps.google.v2.GoogleMapActivity}: java.lang.IllegalStateException: Already attached
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3270)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
     Caused by: java.lang.IllegalStateException: Already attached
        at androidx.fragment.app.FragmentManagerImpl.attachController(FragmentManagerImpl.java:2563)
        at androidx.fragment.app.FragmentController.attachHost(FragmentController.java:116)
        at androidx.fragment.app.FragmentActivity.onCreate(FragmentActivity.java:284)
        at androidx.appcompat.app.AppCompatActivity.onCreate(AppCompatActivity.java:115)
        at cgeo.geocaching.maps.google.v2.GoogleMapActivity.superOnCreate(GoogleMapActivity.java:134)
        at cgeo.geocaching.maps.AbstractMap.onCreate(AbstractMap.java:55)
        at cgeo.geocaching.maps.CGeoMap.onCreate(CGeoMap.java:536)
        at cgeo.geocaching.maps.google.v2.GoogleMapActivity.onCreate(GoogleMapActivity.java:67)
        at android.app.Activity.performCreate(Activity.java:7802)
        at android.app.Activity.performCreate(Activity.java:7791)
[...]
```

Happens on opening live map from the main screen, with Google Maps being the active map provider.

According to some SO articles this comes from `onCreate` etc. being called twice, which I can confirm after some debugging. Looks like being introduced with fixing the lint "missing super calls" messages with #10251, and may be system-specific or only happening in the emulator, as @fm-sys got past this error, crashing on #10281, whereas while debugging that in the emulator, I did not even come that far. (Edit: But he has Mapsforge set as active map provider, so that might be the reason why he did not run into this crash.)

--- a/main/src/cgeo/geocaching/maps/google/v2/GoogleMapActivity.java
+++ b/main/src/cgeo/geocaching/maps/google/v2/GoogleMapActivity.java
@@ -19,6 +19,7 @@ import static cgeo.geocaching.settings.Settings.MAPROTATION_AUTO;
 import static cgeo.geocaching.settings.Settings.MAPROTATION_MANUAL;
 import static cgeo.geocaching.settings.Settings.MAPROTATION_OFF;
 
+import android.annotation.SuppressLint;
 import android.content.Intent;
 import android.os.Bundle;
 import android.view.Menu;
@@ -32,6 +33,7 @@ import androidx.appcompat.app.AppCompatActivity;
 
 import org.apache.commons.lang3.StringUtils;
 
+@SuppressLint("MissingSuperCall") // super calls are handled via mapBase (mapBase.onCreate, mapBase.onSaveInstanceState, ...) TODO: Why is it done like that?
 public class GoogleMapActivity extends AppCompatActivity implements MapActivityImpl, FilteredActivity {
 
 
@@ -63,13 +65,11 @@ public class GoogleMapActivity extends AppCompatActivity implements MapActivityI
 
     @Override
     protected void onCreate(final Bundle icicle) {
-        super.onCreate(icicle);
         mapBase.onCreate(icicle);
     }
 
     @Override
     protected void onSaveInstanceState(@NonNull final Bundle outState) {
-        super.onSaveInstanceState(outState);
         mapBase.onSaveInstanceState(outState);
     }
 
@@ -80,19 +80,16 @@ public class GoogleMapActivity extends AppCompatActivity implements MapActivityI
 
     @Override
     protected void onDestroy() {
-        super.onDestroy();
         mapBase.onDestroy();
     }
 
     @Override
     protected void onPause() {
-        super.onPause();
         mapBase.onPause();
     }
 
     @Override
     protected void onResume() {
-        super.onResume();
         mapBase.onResume();
     }
 
@@ -113,7 +110,6 @@ public class GoogleMapActivity extends AppCompatActivity implements MapActivityI
 
     @Override
     protected void onStart() {
-        super.onStart();
         //Target view
         mapBase.targetView = new TargetView((TextView) findViewById(R.id.target), (TextView) findViewById(R.id.targetSupersize), StringUtils.EMPTY, StringUtils.EMPTY);
         final Geocache target = mapBase.getCurrentTargetCache();
@@ -125,7 +121,6 @@ public class GoogleMapActivity extends AppCompatActivity implements MapActivityI
 
     @Override
     protected void onStop() {
-        super.onStop();
         mapBase.onStop();
     }
 

